### PR TITLE
Chore: Fix the typo in multiple files

### PR DESCRIPTION
**Pull request by thapasusheel**

### CODE REVIEW

**Review by MarkDaoust — APPROVED** [review]
Nice! Thanks!

**Comment by MarkDaoust** [comment]
Oh, why'd you close this? We haven't merged it yet.

I'm reopening for the merge.

**Comment by thapasusheel** [comment]
> Oh, why'd you close this? We haven't merged it yet.
> 
> I'm reopening for the merge.

That was mistakely closed. I am sorry from my side.

**Comment by thapasusheel** [comment]
@MarkDaoust Could you let me know when will this PR be merged?

File: site/en/guide/tf_numpy_type_promotion.ipynb [hunk 1]
```diff
@@ -178,7 +178,8 @@
         "*   `f32*` means Python `float` or weakly-typed `f32`\n",
         "*   `c128*` means Python `complex` or weakly-typed `c128`\n",
         "\n",
-        "The asterik (*) denotes that the corresponding type is “weak” - such a dtype is temporarily inferred by the system, and could defer to other dtypes. This concept is explained more in detail [here](#weak_tensor)."
+        "The asterisk\n",
+        " (*) denotes that the corresponding type is “weak” - such a dtype is temporarily inferred by the system, and could defer to other dtypes. This concept is explained more in detail [here](#weak_tensor)."
```
Comment (8bitmp3) [inline site/en/guide/tf_numpy_type_promotion.ipynb:182]:
```suggestion
        "The asterisk (*) denotes that the corresponding type is “weak” - such a dtype is temporarily inferred by the system, and could defer to other dtypes. This concept is explained more in detail [here](#weak_tensor)."
```

**Review by 8bitmp3 — APPROVED** [review]

File: site/en/guide/sparse_tensor.ipynb [hunk 2]
```diff
@@ -31,6 +31,11 @@
         "# limitations under the License."
       ]
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": []
+    },
```
Comment (8bitmp3) [inline site/en/guide/sparse_tensor.ipynb:38]:
```suggestion
```

**Review by 8bitmp3 — APPROVED** [review]

**Pull request merged**